lib/game.rb

```diff
@@ -1,11 +1,6 @@
 class Game
-  attr_reader :game_id,
-              :season,
-              :type,
-              :away_team_id,
-              :home_team_id,
-              :away_goals,
-              :home_goals
+  attr_reader :game_id, :season, :type, :away_team_id,
+              :home_team_id, :away_goals, :home_goals
```

Comment:
I like the fewer lines

Reply:
Theres fewer lines here now. Nice!

Reply:
I agree. I want to say the correct format is to have each one have it's own line. But for sake of reducing lines.... solid decision.